input/pagecontent/group-genomics.md

```diff
@@ -34,14 +107,21 @@ The mCODE example [`genomic-variant-fusion`](Observation-genomic-variant-fusion.
 
 **NOTE:** A fusion event that is related to two variants will be expressed as two separate variant instances. Both variants from the same fusion event can be included as DiagnosticReport.result slices under [GenomicsReport].
 
-### Profiles
+### Search parameters for genomic variants
+
+The [GRIG provides extensive guidance and examples](http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html#appendix-d-query-guidance) for querying FHIR genomic reporting data. Some of the ones relevant to mCODE are listed below for convenience.
+
+**NOTE:** Genomic operations are out of scope for mCODE STU3 since it is not yet approved for publication at this time.
+
+#### [Search a patient's variants with a given gene](http://build.fhir.org/ig/HL7/genomics-reporting/usecases.html#search-for-variants-with-a-given-gene)
```

Comment:
Don't put a link in a heading. Put it in the text.

Also, shorten the heading to "Search for Variants with Given Gene" (with title capitalization)